openshift/sandboxed-containers-operator
# KATA-3281: Getting RVPS values from SE PODVM

**- Description of the problem which is fixed/What is the use case**
We are trying to retrieve some of the parameters from PODVM SE image which can be used for remote attestation. with this approach we can facilitate to automate the kbs deployment.

**- What I did**
--> Added Logic to sandbox-container-operator to retrieve some of image parameters once it gets created.

**- How to verify it**
--> Create OSC-PODVM-builder image, create job and check the RVPS values is getting extracted or not. 
      After extraction, check whether it has been uploaded on libvirt volume.  

**- Description for the changelog**
